@xen-orchestra/web-core/lib/components/ui/counter/UiCounter.vue

```diff
@@ -128,17 +128,17 @@ const classNames = computed(() => {
   /* SIZE VARIANTS */
 
   &.size--small {
-    height: 1.5rem;
-    min-width: 1.5rem;
-    border-radius: calc(1.5rem / 2);
-    padding: 0 0.4rem;
+    height: 1.6rem;
+    min-width: 1.6rem;
+    border-radius: calc((1.6rem + 0.4rem) / 2);
```

Comment:
```suggestion
    border-radius: 0.8rem;
```
The + 0.4rem does not change anything in the display, since the final radius will always be limited to 0.8rem (height/2)